bindings/julia/Displaz.jl

```diff
@@ -37,28 +37,71 @@ function ply_property_name(semantic, idx)
 end
 
 # Write a set of points to displaz-native ply format
-function write_ply_points(fileName, nvertices, fields)
-    convertedFields = [ply_type_convert(value) for (_,__,value) in fields]
-    open(fileName, "w") do fid
+function write_ply_points(filename, nvertices, fields)
+    converted_fields = [ply_type_convert(value) for (_,__,value) in fields]
+    open(filename, "w") do fid
         write(fid, "ply\n")
         write(fid, "format binary_little_endian 1.0\n")
         write(fid, "comment Displaz native\n")
-        for ((name,semantic,_), (typeName, value)) in zip(fields, convertedFields)
+        for ((name,semantic,_), (typename, value)) in zip(fields, converted_fields)
             n = size(value,1)
             assert(n == nvertices || n == 1)
             write(fid, "element vertex_$name $n\n")
             for i = 1:size(value,2)
-                propName = ply_property_name(semantic, i)
-                write(fid, "property $typeName $propName\n")
+                propname = ply_property_name(semantic, i)
+                write(fid, "property $typename $propname\n")
             end
         end
         write(fid, "end_header\n")
-        for (_,value) in convertedFields
+        for (_,value) in converted_fields
             write(fid, value')
         end
     end
 end
 
+function write_ply_lines(filename, position, color, linebreak)
+    nvalidvertices = size(position,1)
+
+    # Create and write to ply file
+    fid = open(filename, "w")
+    write(fid, 
+        """
+        ply
+        format binary_little_endian 1.0
+        element vertex $nvalidvertices
+        property double x
+        property double y
+        property double z
+        element color $nvalidvertices
+        property float r
+        property float g
+        property float b
+        element edge $(length(linebreak))
+        property list int int vertex_index
+        end_header
+        """
+    )
+
+    write(fid,convert(Array{Float64,2},position'))
```

Comment:
I'm going to assume this is something to do with position sometimes being an array of the wrong type (eg, an integer type).  I'll leave it in for now since I'd like to merge this.  We can probably do something more consistent with the types before passing to `write_ply_*` later.